### PR TITLE
disable ENCRYPTION

### PR DESCRIPTION
**Pull request by mdayican**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/BPS-838

### Change description ###

disable ENCRYPTION for staging

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: charts/rpe-send-letter-service/values.aat.template.yaml [hunk 1]
```diff
@@ -3,3 +3,5 @@ java:
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
   devApplicationInsightsInstrumentKey: "71df5a0f-ebbe-445c-b1d1-d10d349f0970"
+  environment:
+    ENCRYPTION_ENABLED: "false"
```
Comment (nitinprabhu) [inline charts/rpe-send-letter-service/values.aat.template.yaml]: encryption is enabled on AAT ?

Comment (mdayican) [reply charts/rpe-send-letter-service/values.aat.template.yaml]: for staging , in jenkins_cnp encryption is false so it should be false here as well.

Comment (nitinprabhu) [reply charts/rpe-send-letter-service/values.aat.template.yaml]: This is used by the app ? In jenkins file we set for functional tests

Comment (mdayican) [reply charts/rpe-send-letter-service/values.aat.template.yaml]: LetterService is using.

Comment (mdayican) [reply charts/rpe-send-letter-service/values.aat.template.yaml]: in jenkins it is TEST_ENCRYPTION_ENABLED

**Comment by mdayican** [comment]
> encryption should be ON



> encryption should be ON

it should be off as tests are expecting OFF.
in  jenkinsfile_CNP  env.TEST_ENCRYPTION_ENABLED = false

**Pull request merged**